host/xtest/xtest_1000.c

```diff
@@ -411,9 +411,10 @@ static void xtest_tee_test_invalid_mem_access(ADBG_Case_t *c, uint32_t n)
 	TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
 	uint32_t ret_orig;
 
-	(void)ADBG_EXPECT_TEEC_SUCCESS(c,
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
 		xtest_teec_open_session(&session, &os_test_ta_uuid, NULL,
-					&ret_orig));
+		                        &ret_orig)))
+		return;
 
 	op.params[0].value.a = n;
 	op.paramTypes = TEEC_PARAM_TYPES(TEEC_VALUE_INPUT, TEEC_NONE, TEEC_NONE,
```

Comment:
I know it's a bit out of the scope of this pr. Would you mind fixing the identation rigth below. The 2 calls have same args but you must read twice to understand it.